node/src/effect.rs

```diff
@@ -789,6 +789,25 @@ impl<REv> EffectBuilder<REv> {
         .await
     }
 
+    /// Gets missing descendant trie keys for an ancestral trie key.
+    #[allow(unused)]
```

Comment:
eh?

Reply:
The other function for doing `put_trie` is also unused (temporarily).

This effect is less involved, so I'll just shelve this PR for now until I use this function.